### ISSUE TITLE
form preserve-scroll not working

### ISSUE THREAD

**Issue by cobraprojects** [protonemedia/laravel-splade#395]
- Laravel Version: 10 latest
- PHP Version: 8.2
- Splade JS Version (npm):1.4.7
- Splade PHP Version (composer): 1.4.7
- Dev environment (OS, Sail/Valet/etc): valet

### Description:

```
<x-splade-form method="get" action="{{ route('branch.index') }}" preserve-scroll>
    <x-splade-select @change="form.submit" name="filter[city]">
          <option value="">All</option>
          @foreach($cities as $city)
              <option value="{{ $city->id }}">{{ $city->title }}</option>
          @endforeach
      </x-splade-select>
</x-splade-form>
```
On change the form submited succefully but it doesn't preserve scroll
Also the select resets after submit

https://streamable.com/6xwv8g

**Comment by cobraprojects** [comment]
I notice that preserve scroll is work after redirect

**Comment by pascalbaljet** [comment]
Fixed in v1.4.15. Make sure to [update both packages](https://splade.dev/docs/upgrading) (JS + PHP).
